src/collectd.conf.pod

```diff
@@ -8148,6 +8148,41 @@ want to use authentication all three fields must be set.
 
 =back
 
+=head2 Plugin C<write_prometheus>
+
+The I<write_prometheus plugin> implements a tiny webserver that can be scraped
+using I<Prometheus>.
+
+B<Options:>
+
+=over 4
+
+=item B<Port> I<Port>
+
+Port the embedded webserver should listen on. Defaults to B<9103>.
+
+=item B<StalenessDelta> I<Seconds>
```

Comment:
We only discussed GC, not this sort of behaviour. The issue is what you've implemented here will break both now and once we finally fix staleness.

Consider that there's a metric being forwarded through a few systems, so that it's hovering around the 5m old mark when it gets to Prometheus. Every time it goes over 5m, Prometheus will see it as now. As Prometheus only allows you to append samples in order, this means the next 5 minutes of samples under 5 minutes old will be rejected and errors logged.

To avoid this you should always expose timestamps.